packages/site/src/utils/snap.ts

```diff
@@ -26,6 +26,17 @@ export const connectSnap = async (
     version: snapPackageInfo.version,
   },
 ) => {
+  // check for current connected chain and force user to switch to sepolia.
+
+  const currentChain = window.ethereum.networkVersion;
+  if (currentChain !== '11155111') {
```

Comment:
we have to force users to connect to 'Boba Sepolia' instead

Reply:
aah I see will update!